bindings/python/pymongoarrow/__init__.py

```diff
@@ -29,14 +29,3 @@ def _parse_version(version):
     raise ImportError(
         "Expected libbson version {} or greater, found {}}".format(
             _MIN_LIBBSON_VERSION, libbson_version))
-
-
-from pymongoarrow.api import aggregate_arrow_all, find_arrow_all
-from pymongoarrow.schema import Schema
-
-
-__all__ = [
-    'aggregate_arrow_all',
-    'find_arrow_all',
-    'Schema'
-]
```

Comment:
Why remove this?

Reply:
Defining `__all__` here was defining the public API for the entire module. This was causing PyCharm to basically throw warnings for every single import in the project because it construed it as me trying to use non-public members of all submodules.

Reply:
I think eventually we will need to define some sensible `__all__` here but it can wait while the project is still under development.

Reply:
Interesting, thanks for explaining.